code/modules/projectiles/guns/projectile/automatic/ak47.dm

```diff
@@ -24,7 +24,7 @@
 	cocked_sound 	= 'sound/weapons/guns/interact/ltrifle_cock.ogg'
 	recoil_buildup = 1.8
 	one_hand_penalty = 15 //automatic rifle level
-	damage_multiplier = 3
+	damage_multiplier = 1.6
```

Comment:
```suggestion
	damage_multiplier = 1.2
```

Reply:
Changed multiplier in commit below. Also set fs ak multiplier to 1